HospitalToday/UI/MenuBuilder.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalToday.UI
+{
+    class MenuBuilder
+    {
+        public MenuBuilder()
+        {
+            menuBuilder = new StringBuilder();
+            menuData = new MenuData();
+        }
+
+        private StringBuilder menuBuilder;
+        private MenuData menuData;
+
+        public int MainMenu()
+        {
+            Clear();
+            menuData.MainMenuData.ForEach(x => menuBuilder.AppendLine(x));
+            Console.WriteLine(menuBuilder);
+            int req;
+            Console.WriteLine("Answer:");
+            var answer = Console.ReadLine();
+            var result = int.TryParse(answer, out req);
+            
+            return result ? req : -1;
+        }
+
+        public int DoctorMenu()
+        {
+            Clear();
+            menuData.DoctorsMenuData.ForEach(x => menuBuilder.AppendLine(x));
+            Console.WriteLine(menuBuilder);
+            int req;
+            Console.WriteLine("Answer:");
```

Comment:
Minor suggestion:
it is better to separate lines by logic

Reply:
Added indents to separate logic structure.